packages/codebytes/src/languageSelection.tsx

```diff
@@ -0,0 +1,46 @@
+import { Select } from '@codecademy/gamut';
+import { Background, themed } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import React from 'react';
+
+import type { languageOption } from './consts';
+import { languageOptions } from './consts';
+
+const StyledSelect = styled(Select)`
+  margin-top: 1rem;
+  color: ${themed('colors.text')};
```

Comment:
oh a third way to get a color haha, is this the preference now? (instead of importing colors or theme?)

Reply:
this seems like an alternative way to get themed colors that may have been in vogue when codebytes was created, updating